dev/VSIX/ProjectTemplates/Desktop/CSharp/PackagedApp/WinUI.Desktop.Cs.PackagedApp.csproj

```diff
@@ -0,0 +1,72 @@
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>WinUI.Desktop.Cs.PackagedApp</RootNamespace>
+    <AssemblyName>WinUI.Desktop.Cs.PackagedApp</AssemblyName>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <GeneratePkgDefFile>false</GeneratePkgDefFile>
+    <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
+    <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
+    <IncludeDebugSymbolsInLocalVSIXDeployment>false</IncludeDebugSymbolsInLocalVSIXDeployment>
+    <CreateVsixContainer>false</CreateVsixContainer>
+    <DeployExtension>false</DeployExtension>
+    <DeployVSTemplates>false</DeployVSTemplates>
+    <CopyVsixManifestToOutput>false</CopyVsixManifestToOutput>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.VisualStudio.CoreUtility">
+      <Private>False</Private>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="WinUI.Desktop.Cs.PackagedApp.ico" />
+    <Content Include="WinUI.Desktop.Cs.PackagedApp.png" />
+    <VSTemplate Include="WinUI.Desktop.Cs.PackagedApp.vstemplate" />
+    <LinkedVSTemplate Include="BlankApp\WinUI.Desktop.Cs.BlankApp.vstemplate" />
+    <LinkedVSTemplate Include="WapProj\WinUI.Desktop.Cs.WapProj.vstemplate" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
+  <ItemGroup>
+    <None Include="BlankApp\Properties\PublishProfiles\WinArm64.pubxml" />
+    <None Include="BlankApp\Properties\PublishProfiles\WinX64.pubxml" />
+    <None Include="BlankApp\Properties\PublishProfiles\WinX86.pubxml" />
+    <None Include="BlankApp\ProjectTemplate.csproj" />
+    <None Include="WapProj\WapProjTemplate.wapproj" />
+  </ItemGroup>
```

Comment:
></ItemGroup> [](start = 2, length = 12)

we need to copy Microsoft.WinUI.AppX.targets for InApp support

Reply:
Microsoft.WinUI.AppX.targets now comes from the transport package

Reply:
the original comment suggests copy is to avoid nuget reference from wap. is it no longer the case, or what's differing this VSIX from the WinUI VSIX?

Reply:
To fix the AppxPackageRegistration issue, I added a direct import to Microsoft.ProjectReunion.AppXReference.props, which assumes the Microsoft.ProjectReunion nupkg is installed for the solution.  I adopted the same approach for Microsoft.WinUI.AppX.targets, importing it directly, assuming the Microsoft.ProjectReunion.WinUI nupkg is installed.  WAP projects now support package references, so it makes sense to move to this approach.